src/main/java/rockset/AvroParser.java

```diff
@@ -56,7 +56,50 @@ private boolean isLogicalType(Schema schema) {
     return LOGICAL_TYPE_CONVERTERS.containsKey(schema.name());
   }
 
-  private Map<String, Object> convertLogicalTypes(Schema valueSchema, Map<String, Object> map) {
+  private boolean isScalarType(Schema schema) {
+    return (schema.type() == Type.STRING ||
```

Comment:
How about defining a static set of SCALAR_TYPES and doing a similar lookup to the above method?

Reply:
On second thoughts, I think we can get rid of this method altogether. If its not logical type and complex type, we can assume it's a scalar type, and use value as-is.